interfaces/builtin/home_test.go

```diff
@@ -123,6 +124,14 @@ func (s *HomeInterfaceSuite) TestUnexpectedSecuritySystems(c *C) {
 	c.Assert(snippet, IsNil)
 }
 
-func (s *HomeInterfaceSuite) TestAutoConnect(c *C) {
+func (s *HomeInterfaceSuite) TestAutoConnectOnClassic(c *C) {
```

Comment:
Please note this must be done on both ends, otherwise this test will break when running under a native env.